fe/fe-core/src/main/java/org/apache/doris/nereids/rules/analysis/CheckBound.java

```diff
@@ -46,6 +51,13 @@ public List<Rule> buildRules() {
                     checkBound(plan);
                     return null;
                 })
+            ),
+            RuleType.CHECK_BITMAP_ANALYSIS.build(
+                logicalHaving().thenApply(ctx -> {
+                    LogicalHaving<Plan> having = ctx.root;
+                    checkHavingBitmapExpression(having);
+                    return null;
+                })
```

Comment:
add comment about this rule to explain why we need this check and what will happen if we do not have this rule